local/gugcat/locallib.php

```diff
@@ -664,7 +664,8 @@ public static function switch_display_of_assessment_on_student_dashboard($instan
                 else{
                     if(!empty($customfieldfield)){
                         $customfieldddata = self::default_contextfield_data_value($customfieldfield->id, $instanceid, $contextid);
-                        $DB->insert_record('customfield_data', $customfieldddata);
+                        $customfielddataid = $DB->insert_record('customfield_data', $customfieldddata, $returnid=true, $bulk=false);
```

Comment:
you can remove "returnid = true" and "$bulk=false", it is a default value so no need to include it.